src/utils/getCalendarMonthWeeks.js

```diff
@@ -1,39 +1,39 @@
-export default function getCalendarMonthWeeks(month, enableOutsideDays) {
+import moment from 'moment';
+
+export default function getCalendarMonthWeeks(
+  month,
+  enableOutsideDays,
+  firstDayOfWeek = moment.localeData().firstDayOfWeek(),
```

Comment:
I thought about that, but I didn't see any other example of this kind of validation. Even here we call `month.clone()` without knowing if `month` is in fact a moment object. Moreover, like all these parameters of `utils` functions, values came from component props that already pass by some `propTypes` validation in dev.